backend/testfiles/execution/stdlib/list.dark

```diff
@@ -126,7 +126,10 @@ Builtin.List.findFirst [ 1; -33; 3; -2; 12 ] (fun x -> (x < 0 && x % 2 == 0)) =
 // PACKAGE.Darklang.Stdlib.List.flatten_v0 [1;2;3] =
 //   Builtin.Test.runtimeError "In List.flatten's 1st argument (`list`), the value should be a List<List<'a>>. However, a List<Int> ([1; 2; 3]) was passed instead.\n\nExpected: List<List<'a>>\nActual: List<Int>: [1; 2; 3]"
 PACKAGE.Darklang.Stdlib.List.flatten_v0 [ [ 1 ]; [ 2 ]; [ 3 ] ] = [ 1; 2; 3 ]
-PACKAGE.Darklang.Stdlib.List.flatten_v0 [ [ 1 ]; [ [ 2; 3 ] ] ] = [ 1; [ 2; 3 ] ]
+
+PACKAGE.Darklang.Stdlib.List.flatten_v0 [ [ 1 ]; [ [ 2; 3 ] ] ] = Builtin.Test.derrorMessage
+  "Could not merge types List<Known (KTList (Known (KTList (Known KTInt))))> and List<Known (KTList (Known KTInt))>"
```

Comment:
this is bad, but I can't think of a way to get this done without growing the scope of this PR greatly or temporarily including a local toString... actually, why don't I do that?